docker-compose.yml

```diff
@@ -1,22 +1,32 @@
 version: "3.9"
-# We can replace DB service later if needed with another one
 services:
+  mysql:
+    image: mysql:8.0
```

Comment:
If the image is not working on Mac computers with the M1 chip, you can try the following alternative:
```
    image: mysql/mysql-server:8.0.32
```

If this modification works on both Mac and Windows, we can proceed with this change.